src/common/config/index.js

```diff
@@ -324,6 +327,34 @@ export default class Config extends EventEmitter {
     return newTeams;
   }
 
+  /**
+   * Apply a default sort order to the team list, if no order is specified.
+   * @param {array} teams to sort
+   */
+  sortUnorderedTeams(teams) {
+    // Make a best pass at interpreting sort order. If an order is not specified, assume it is 0.
+    //
+    const newTeams = teams.sort((x, y) => {
+      if (x.order == null) {
+        x.order = 0;
+      }
+      if (y.order == null) {
+        y.order = 0;
+      }
+      return (x.order === y.order) ? 0 : ((x.order > y.order) ? 1 : -1); // eslint-disable-line no-nested-ternary
```

Comment:
Sorting numbers can be efficiently done by simply substracting.
```suggestion
      // once we ensured `order` exists, we can sort numerically
      return x.order - y.order;
```